### PR TITLE
[PLATFORM-961] Refactor responsive menu

### PR DESCRIPTION
**Pull request by juhah**
Changes core responsive menu to match design:

**Signed out** 

Design
Screen Shot 2020-03-05 at 17 00 37

Implementation
Screen Shot 2020-03-05 at 17 20 26

https://share.goabstract.com/34d27f84-98cb-44b4-9f3a-fc668bc91937?mode=design&sha=8e5e8db7de798d4970de1594ea92ea8f22776f34

**Signed in**

Design
Screen Shot 2020-03-05 at 17 02 21

Implementation
Screen Shot 2020-03-05 at 17 20 08

https://share.goabstract.com/f80f3c4f-9807-405d-a4dd-c1b08f1fe75f?mode=design&sha=8e5e8db7de798d4970de1594ea92ea8f22776f34

Also added truncation of of ETH address username in desktop menu:
Screen Shot 2020-03-05 at 16 57 34

### CODE REVIEW

File: app/src/shared/components/MobileNav/Bar/bar.pcss [hunk 1]
```diff
@@ -2,7 +2,7 @@
   background-color: white;
   display: flex;
   justify-content: space-between;
-  padding: calc(1.25 * var(--um)) calc(1.5 * var(--um));
+  padding: calc(1.25 * var(--um)) var(--um);
```
Comment (juhah) [inline app/src/shared/components/MobileNav/Bar/bar.pcss]: Tightened the padding here to align the hamburger & close icon when toggling:
Screen Shot 2020-03-05 at 17 20 03

**Comment by juhah** [comment]
> Seems like you're using Sans & perhaps the wrong size throughout those menus, which are in Mono & a different size.

@mattatgit Thanks, so it was - fixed in https://github.com/streamr-dev/streamr-platform/pull/915/commits/a2a9c996649371441c17741545215c2e0c4dbd02:

Screen Shot 2020-03-06 at 10 42 13

Screen Shot 2020-03-06 at 10 42 32

File: app/src/shared/components/MobileNav/mobileNav.pcss [hunk 2]
```diff
@@ -26,11 +26,47 @@
   visibility: visible;
 }
 
+.userWrapper {
+  padding: 0 calc(1.5 * var(--um));
```
Comment (mondoreale) [inline app/src/shared/components/MobileNav/mobileNav.pcss]: `0 1.5um`

File: app/src/shared/components/MobileNav/mobileNav.pcss [hunk 3]
```diff
@@ -26,11 +26,47 @@
...
+  display: grid;
+  grid-template-columns: 40px 1fr;
+  grid-column-gap: var(--um);
```
Comment (mondoreale) [inline app/src/shared/components/MobileNav/mobileNav.pcss]: `1um`

File: app/src/shared/components/MobileNav/mobileNav.pcss [hunk 4]
```diff
@@ -26,11 +26,47 @@
...
+
+  .name {
+    font-weight: var(--medium);
+    font-size: var(--um);
+    line-height: 20px;
+  }
+
+  .username {
+    color: #A3A3A3;
+    font-size: calc(0.75 * var(--um));
```
Comment (mondoreale) [inline app/src/shared/components/MobileNav/mobileNav.pcss]: `0.75um`, and so on.

**Pull request merged**